src/Serilog/ReflectionTypesScalarDestructuringPolicy.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Serilog.Policies
+{
+    class ReflectionTypesScalarDestructuringPolicy : IDestructuringPolicy
+    {
+        public bool TryDestructure(object value, ILogEventPropertyValueFactory propertyValueFactory, out LogEventPropertyValue result)
+        {
+            var type = value as System.Type;
```

Comment:
Force-pushed in 29c14d0

![giphy](https://cloud.githubusercontent.com/assets/570470/14938795/701041d8-0f73-11e6-807b-932694fc1d1f.gif)